src/formio/components/Cosign.stories.js

```diff
@@ -0,0 +1,40 @@
+import {withUtrechtDocument} from 'story-utils/decorators';
+
+import {SingleFormioComponent} from './story-util';
+
+export default {
+  title: 'Form.io components / Custom / Cosign',
+  decorators: [withUtrechtDocument],
+  args: {
+    key: 'cosign',
+    type: 'cosign',
+    label: 'Co-signer email address',
+    validateOn: 'blur',
+    authPlugin: 'digid',
+    extraComponentProperties: {},
+    evalContext: {},
+  },
```

Comment:
I think that maybe it is not necessary to specify the args here (and you can just specify them in the story). :thinking: 

If you remove them, you will need to add the `type` in the args of the story below.